donut/static/js/groups/positions.js

```diff
@@ -214,14 +261,31 @@ $(document).ready(function() {
             }
         });
     });
+    $('#submitDelPosHoldBtn').click(function(e) {
+        e.preventDefault();
```

Comment:
Should be able to avoid calling `preventDefault()` if you add the attribute `type = 'button'` to the button, so it doesn't act as a submission button

Reply:
Ok, it makes sense to do this for the other buttons as well